.vsts-dotnet-ci.yml

```diff
@@ -100,7 +100,12 @@ jobs:
 - job: FullReleaseOnWindows
   displayName: "Windows Full Release (no bootstrap)"
   pool:
-    vmImage: 'windows-2022'
+    ${{ if eq(variables['System.TeamProject'], 'public') }}:
+      name: NetCore-Public
```

Comment:
This is the Windows _Full_ build, so NetCore seems...not right? But on the other hand, the ImageOverride suggests it's a vs preview version, so that would presumably be full framework. I'm a bit confused here.

Reply:
`NetCore-Public` is the name of the [pool](https://learn.microsoft.com/azure/devops/pipelines/agents/pools-queues) maintained by dnceng for use by the .NET repos. Within it there are [many machine types](https://helix.int-dot.net/#1ESHostedPoolImagesWestUS-Int-rg-Public-Windows), including the Windows + preview VS one I'm specifying below.